backend/utils/email_utils.py

```diff
@@ -48,6 +54,29 @@ def is_valid_email(email: str) -> bool:
         print(str(e))
         return False
 
+def get_email_content(email_data: EmailData) -> str:
```

Comment:
Why not just use the value in the data type definition of the function? Instead of adding global variable at the top
```suggestion
def get_email_content(email_data: Dict[str, Any]) -> str:
```

Reply:
Experimenting with letting Copilot write my typehints haha. Like your way better.